app/Services/UserService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Storage;
+
+use App\Models\User;
+
+class UserService
+{
+    public function create($request)
+    {
+        $avatarUploadedPath = null;
+        if ($request->file('avatar')) {
+            $uploadAvatarFolder = 'users';
+            $avatar = $request->file('avatar');
+            $avatarUploadedPath = $avatar->store($uploadAvatarFolder, 'public');
+            $uploadedImageResponse = [
+            'image_name' => basename($avatarUploadedPath),
+            'image_url' => Storage::disk('public')->url($avatarUploadedPath),
+            ];
```

Comment:
Sao cái này CI ko báo lỗi nhỉ =))
```suggestion
            $uploadedImageResponse = [
                'image_name' => basename($avatarUploadedPath),
                'image_url' => Storage::disk('public')->url($avatarUploadedPath),
            ];
```

Reply:
Dạ chắc CI thấy code chán quá nên cho qua (J4F)
Mà em sẽ để ý hơn ạ, để a chỉ cần review Tech thôi. (facepalm) (sr)